### DISCUSSION TITLE
Ajouter une question pour les charges non deductibles pour les entreprises

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2605] [Idées]
Certaines charges sont non déductibles. Si on veut faire une simulation complète de l'impôt, il faudrait pouvoir spécifier un montant de charge non déductible

C'est aussi très utile pour les auto-entrepreneurs. Exemple de retour : 

> Le résultat est fortement biaisé, il ne prend pas en compte toutes les dépenses liées aux métiers pratiqués ainsi qu'aux investissements nécessaires, réguliers et indispensables. Mes revenus sont dans les faits plus de trois fois plus faibles que ce que vous annoncez...

https://www.service-public.fr/professionnels-entreprises/vosdroits/F31973

**Comment by mquandalle** [comment]
On a eu le retour pour les auto-entrepreneurs :

>  Il serait intéressant de mettre un onglet pour les charges et dépenses liées à l'activité.

**Comment by wiinxt** [comment]
Un nouveau retour pour ajouter les charges pour les médecin aussi :

> Il serait utile de pouvoir saisir, même approximativement, en fin de calcul et en facultatif, les autres dépenses (Véhicule, cabinet, diverses cotisations, assurance, secrétariat, matériel...) pour avoir au final le résultat net de l'argent dont le médecin dispose.

**Comment by johangirod** [comment]
@wiinxt, je pense que c'est différent. Il est déjà possible de déduire les dépenses déductibles dans les simulateurs indépendants, il suffit de les préciser dans la case `charges`. Visiblement, cette personne n'a pas compris cela.

**Comment by wiinxt** [comment]
J'avais été voir et il n'y avais pas de question sur les charges, j'avais pas fait gaffe que c'était dans le bloc bleu 
Du coup je vais lui dire
